infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogReplicationSourceManager.java

```diff
@@ -73,20 +72,6 @@ public LogReplicationSourceManager(LogReplicationRuntimeParameters params,
                                        LogReplicationMetadataManager metadataManager, DataSender dataSender,
                                        LogReplicationSession session, LogReplicationContext replicationContext) {
 
-        // This runtime is used exclusively for the snapshot and log entry reader which do not require a cache
-        // as these are one time operations.
-        this.runtime = CorfuRuntime.fromParameters(CorfuRuntimeParameters.builder()
-                .trustStore(params.getTrustStore())
-                .tsPasswordFile(params.getTsPasswordFile())
-                .keyStore(params.getKeyStore())
-                .ksPasswordFile(params.getKsPasswordFile())
-                .systemDownHandler(params.getSystemDownHandler())
-                .tlsEnabled(params.isTlsEnabled())
-                .cacheDisabled(true)
-                .maxWriteSize(params.getMaxWriteSize())
-                .build());
-        runtime.parseConfigurationString(params.getLocalCorfuEndpoint()).connect();
```

Comment:
This connect call seems missing. Is it taken care by this change somewhere?

Reply:
no its not missing...its already here: https://github.com/CorfuDB/CorfuDB/blob/log_replication_v2/infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java#L382